lib/main.dart

```diff
@@ -10,6 +11,9 @@ import 'constants/supabase.dart';
 
 Future<void> main() async {
   WidgetsFlutterBinding.ensureInitialized();
+  await Hive.initFlutter();
+  await Hive.openBox(
```

Comment:
According to the [docs](https://docs.hivedb.dev/#/?id=read-amp-write), openbox creates a container to write and read data. We don't need to read/write any data at initialization.